src/devices/LedDisplay/samples/Large4Digit7SegmentDisplay.sample.cs

```diff
@@ -0,0 +1,195 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Device.I2c;
+using System.Threading;
+using Iot.Device.LedDisplay;
+using Iot.Device.LedDisplay.Enums;
+using static Iot.Device.LedDisplay.Ht16k33;
+using static Iot.Device.LedDisplay.Large4Digit7SegmentDisplay;
+
+namespace SevenSegmentSample
+{
+    class Program
+    {
+        const string SupportedCharacters = "0123456789aAbBcCdDeEfFgGhHiIjJlLnNoOpPrRsStuUyYzZ-_|°     ";
+
+        static void Main(string[] args)
+        {
+            // Initialize display (busId = 1 for Raspberry Pi 2 & 3)
+            var display = new Large4Digit7SegmentDisplay(I2cDevice.Create(new I2cConnectionSettings(busId: 1, DefaultI2cAddress)))
+            {
+                // Set max brightness
+                Brightness = 15
```

Comment:
that's exactly why I think this should be `Brightness = 1.0` or `Brightness = Large4Digit7SegmentDisplay.MaxBrightness` - otherwise it's hard to reason about this code and why 15 is max